content/en/tracing/trace_collection/library_injection.md

```diff
@@ -93,6 +93,8 @@ The available library versions are listed in each container registry.
 
 **Note**: If you already have an application instrumented using version X of the library, and then use library injection to instrument using version Y of the same tracer library, the tracer does not break. Rather, the library version loaded first is used. Because library injection happens at the admission controller level prior to runtime, it takes precedent over manually configured libraries.
 
+INSERT LINK TO VARIOUS REPOS FOR TRACING CLIENT
+
```

Comment:
```suggestion
```

Reply:
@Warren3225 Let's go ahead and remove this from the PR. I don't want to delay your other update getting merged. We can open another PR for this once we figure out what we want to do.